python/tests/compat/conftest.py

```diff
@@ -0,0 +1,142 @@
+import pytest
+import subprocess
+import os
+import venv
+import tempfile
+
+class ErrorInVenv(Exception):
+    """To signal errors occuring from within the venv"""
+
+def run_shell_commands(commands_list, cwd=None):
```

Comment:
`commands_list` seems to always have a single element?

Reply:
Fixed